synapse/lib/trigger.py

```diff
@@ -164,18 +224,18 @@ def mod(self, iden, query):
         if rule is None:
             raise s_exc.NoSuchIden()
 
-        if self.enabled:
-            self.core.getStormQuery(query)
+        self.core.getStormQuery(query)
 
-        db = self.core.slab.initdb(self.TRIGGERS_DB_NAME)
         rule.storm = query
-        self.core.slab.put(iden, rule.en(), db=db)
+        self.core.slab.put(iden, rule.en(), db=self.trigdb)
 
     @contextlib.contextmanager
     def _recursion_check(self):
+
         depth = RecursionDepth.get()
-        if depth > self.RECURSION_LIMIT:
+        if depth > 64:
```

Comment:
this changed from 16 to 64 deep. why?

Reply:
Any issues raised by being "too deep" are no different between 16 and 64, while it's way more likely ( and even ultimately, fairly likely ) to hit this limit at 16.